### PR TITLE
updated configs with example values

### PR DESCRIPTION
**Pull request by Rhia2**
## Test plan

No review required

### CODE REVIEW

File: schema/site.schema.json [hunk 1]
```diff
@@ -2068,7 +2322,20 @@
           "type": "integer",
           "minimum": 0
         }
-      }
+      },
+      "examples": [
+        {
+          "summary": "Enable OpenAI embeddings",
```
Comment (indradhanush) [inline schema/site.schema.json:2328]: I don't see a summary property defined above? 🤔

File: schema/site.schema.json [hunk 2]
```diff
@@ -2068,7 +2322,20 @@
...
+          "value": {
```
Comment (indradhanush) [inline schema/site.schema.json:2329]: This should not be here either? And the elements inside this object should be on the top level instead?

**Review by indradhanush — COMMENTED** [review]
Great PR! Thank you for doing this!

Left a few comments.

File: schema/site.schema.json [hunk 3]
```diff
@@ -98,7 +112,17 @@
...
-      }
+      },
+      "examples": [
+        {
+          "auth.passwordPolicy": {
+            "enabled": true,
+            "numberOfSpecialCharacters": 1,
+            "requireAtLeastOneNumber": true,
+            "requireUpperandLowerCase": true
+          }
```
[the remark this answers is not in the document]
Comment (Rhia2) [reply schema/site.schema.json:123]: True.. Thanks for catching that

File: schema/site.schema.json [hunk 4]
```diff
@@ -928,7 +1012,15 @@
...
           "minimum": 1
         }
-      }
+      },
+      "examples": [
+        {
+          "maxTimeoutSeconds": 120,
+          "maxRepos": 500,
+          "commitDiffMaxRepos": 100,
+          "commitDiffWithTimeFilterMaxRepos": 10000
```
[the remark this answers is not in the document]
Comment (Rhia2) [reply schema/site.schema.json]: This has now been changed

File: schema/site.schema.json (hunk 2 above)
Comment (Rhia2) [reply schema/site.schema.json]: Fixed

**Review by indradhanush — APPROVED** [review]
LGTM!

**Pull request closed without merging**